client/signup/steps/domains/index.jsx

```diff
@@ -145,6 +148,34 @@ class DomainsStep extends Component {
 		return isPlansStepExistsInFutureOfFlow && ! isPlanStepSkipped;
 	};
 
+	componentDidMount() {
+		const signupFlows = [
+			'onboarding',
+			'free',
+			'personal',
+			'premium',
+			'business',
+			'ecommerce',
+			'with-theme',
+			'personal-monthly',
+			'premium-monthly',
+			'business-monthly',
+			'ecommerce-monthly',
+			'with-design-picker',
+		];
+		if ( signupFlows.includes( this.props.flowName ) ) {
+			loadExperimentAssignment( 'domain_step_copy_test_202201' ).then( ( experimentName ) => {
```

Comment:
An issue here is that the domain step will show a blank screen while it waits for the API request of `loadExperimentAssignment()` to complete. 



![domaincopytest](https://user-images.githubusercontent.com/1269602/148965242-8225bdb0-bd2e-407d-8cae-56452675d3a4.gif)



We could either show a [PulsingDot](https://github.com/Automattic/wp-calypso/blob/f2e6f88ae65c8571bbac21b284c29b48ec987598/client/components/pulsing-dot/index.jsx) (e.g. usage [here](https://github.com/Automattic/wp-calypso/blob/f2e6f88ae65c8571bbac21b284c29b48ec987598/client/layout/loader.jsx#L12) but modify to use the loading state of the experiment), or my preferred method is to prefetch the assignment in client/signup/controller.js, example usage [here](https://github.com/Automattic/wp-calypso/pull/59771/files#diff-9d7d6e5b810ec4d930d0dd4ab43ab0a72e8082274e818ed6888ec938a94dacc2R330).

Reply:
If you're planning to pre-fetch the assignment, then it's crucial to define an exposure event.

Reply:
Nice, I like that. I switched to your pre-fetch method and added the `domains` step as the exposure event.